project/firstproject/src/main/java/com/example/firstproject/controller/AuthenticationController.java

```diff
@@ -0,0 +1,48 @@
+package com.example.firstproject.controller;
+
+import com.example.firstproject.model.User;
+import com.example.firstproject.services.AuthenticationService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@Controller
+@RequestMapping(value = "/auth")
```

Comment:
Change the route to be **@RequestMapping(value = "/register")** if you create new controller for login and remove register from other routes names:
Example:  `@GetMapping("/register/register-page")` to ` @GetMapping("/register-page")`